subsys/bluetooth/host/gatt.c

```diff
@@ -3464,7 +3464,10 @@ static uint8_t disconnected_cb(const struct bt_gatt_attr *attr, uint16_t handle,
 bool bt_gatt_is_subscribed(struct bt_conn *conn,
 			   const struct bt_gatt_attr *attr, uint16_t ccc_type)
 {
-	const struct _bt_gatt_ccc *ccc;
+	uint16_t ccc_bits;
+	uint8_t properties;
```

Comment:
`uint8_t properties;` can be move to the `if (!bt_uuid_cmp(attr->uuid, BT_UUID_GATT_CHRC)) {` clause to reduce scope of the variable